examples/iot/esp32/voice_led_controller/README.md

```diff
@@ -0,0 +1,104 @@
+# Voice-Controlled LED Ring with the ESP32
```

Comment:
Optional, but does the demo need to be this deep (directory-wise)?

We try to keep things pretty flat but if you're going to build out other iot, esp, etc demos then keep as it is.

Reply:
Totally fair - I'm planning some more esp32 examples and raspberry pi, so figured I'd set the structure up for that up front since each will have their own readme and wiring diagrams.